### ISSUE TITLE
Condition breakpoint not work

### ISSUE THREAD

**Issue by ser-vasilich** [intellij-rust/intellij-rust#5274]
## Environment

* **IntelliJ Rust plugin version:** 0.2.120.2202-201
* **Rust toolchain version:** 1.44.0-nightly (b543afca9 2020-04-05) x86_64-apple-darwin
* **IDE name and version:** CLion 2020.1 (CL-201.6668.126)
* **Operating system:** macOS 10.15.4

## Problem description
Then set condition breakpoint:
1) not work condition breakpoint;
2) not work any breakpoint.

## Steps to reproduce
```rust
fn main() {
    for i in 0..5 {
        println!("Hello!");
    }
}
```
breakpoint in row with println!
set condition i==2

<img width="1643" alt="Screenshot 2020-04-23 at 17 21 14" src="https://user-images.githubusercontent.com/23247647/80109984-dabaab80-8586-11ea-9e49-17457ccc7f33.png">

**Comment by Undin** [comment]
@ser-vasilich Could you please enable `#com.jetbrains.cidr.execution.debugger` debug log setting and attach log file here?
See the corresponding [instruction](https://intellij-support.jetbrains.com/hc/en-us/articles/360001436079-Collecting-additional-logs-in-CLion-)

**Comment by ser-vasilich** [comment]
[idea.log](https://github.com/intellij-rust/intellij-rust/files/4527985/idea.log)

**Comment by ortem** [comment]
I've managed to reproduce the bug with conditional breakpoints.

@ser-vasilich Do simple (not conditional) breakpoints work properly in your environment?

**Comment by ser-vasilich** [comment]
Yes. Simple breakpoints work properly.

**Comment by drewm1980** [comment]
I encountered this (in a more complicated scenario) in 

CLion 2020.3.2
Rust plugin 0.3.140.3644-203

Is this a bug, or is this just not implemented yet?

**Comment by drewm1980** [comment]
I added a ticket against CLion:

https://youtrack.jetbrains.com/issue/CPP-23974?clearDraft=true&description=%0A%0ACL-203.7148.70,%20JRE%2011.0.9.1%2B11-b1145.77x64%20JetBrains%20s.r.o.,%20OS%20Linux(amd64)%20v5.4.0-62-generic,%20screens%201920.0x1200.0

**Comment by ortem** [comment]
@drewm1980 It is a bug. Conditional breakpoints for primitive values should generally work except some specific cases, e.g. loop variables. 

Please ensure that conditional breakpoints work for primitive values of local variables declared at the top level of the function body block (but not inside the loop iteration/condition or something like that). If they don't work even in such simple cases, please let us know in this issue.

**Comment by bravit** [comment]
The following code results in SIGSEGV on M1 when I try to use a conditional breakpoint:

```rust
fn main() {
    let mut idx: i32 = 0;
    while idx < 10 {
        println!("{}", idx);
        idx += 1; // Break at `idx == 5`
    }
}
```

**Comment by ortem** [comment]
Fixed in CLion 2023.1 EAP 231.6471.1

**Comment by 52617365** [comment]
Currently running into this bug when I'm trying to set a conditional breakpoint on a variable inside of a loop.

I'm using CLion with a Rust project.

My breakpoint condition looks like
`instruction_count == 3`

please tell me how I can help more, eg provide certain logs privately or something.
